config/routes.rb

```diff
@@ -51,9 +51,7 @@ def put(*_args); end
       end
       patch '/sources/:source_id/refresh', :to => "sources#refresh", :as => 'refresh'
 
-      resources :tags, :only => [:index, :show] do
-        resources :service_inventories, :only => [:index]
-      end
+      resources :tags, :only => [:index]
```

Comment:
@hsong-rh Just wanted to confirm that this is only direct access to tags but we would still be able to add/remove tags based on the service_inventory. I see them as "/service_inventories/{id}/tag", "/service_inventories/{id}/untag"
"/service_inventories/{id}/tags"

Reply:
yes, adding/removing/listing tags on a service inventory are still there. Only `/tags/{id}/service_inventories` and `/tags/{id}` are removed.